index.js

```diff
@@ -1,13 +1,18 @@
 require('dotenv-safe').config(); // Set up environment variables
 const fetch = require('node-fetch'); // Import fetch (for network requests)
+const moment = require('moment');
 
-// This function (for now) will be what is called when the program is run.
-// Use this to help test other functions
-function main() {
-  console.log('Currently there is nothing here!');
-}
+const EnphaseUtils = require('./utils/enphase.js');
 
-// Add other functions below that perform the individual operations
-// Example: (Pull data from EnPhase API, Generate PDF given billing data, etc.)
+async function main() {
+  console.log('Testing Enphase Data');
+  const result = await EnphaseUtils.getEnphaseData(
+    process.env.SAMPLE_ENPHASE_USER_ID,
+    process.env.SAMPLE_ENPHASE_SYSTEM_ID,
```

Comment:
is there any (dummy) inputs for `SAMPLE_ENPHASE_USER_ID` and `SAMPLE_ENPHASE_SYSTEM_ID` i can call this function with just to verify functionality?

Reply:
Posted a sample environment file in slack!